docs/data/migration/migration-tree-view-lab/migration-tree-view-lab.md

```diff
@@ -0,0 +1,53 @@
+---
+productId: x-tree-view
+---
+
+# Migration from the lab
+
+<p class="description">MUI Tree View is now available on MUI X!</p>
+
+## Introduction
+
+This is a reference for migrating your site's tree view from `@mui/lab` to `@mui/x-tree-view`.
+This migration is about the npm packages used, it **does not** affect the behavior of the components in your application.
+
+[//]: # 'You can find why we are moving in this direction in the [announcement blog post](/blog/lab-tree-view-to-mui-x/).'
```

Comment:
Will uncomment once the blog post is released
This page links to the blog post and the blog post links to this page so I must do it in steps.